documentation/slate/source/includes/apis.md

```diff
@@ -1019,7 +1019,6 @@ See how disruptions affect a journey in the [real time](#realtime) section.
 | nop     | max_nb_transfers      | int     | Maximum number of transfers in each journey  | 10          |
 | nop     | min_nb_transfers     | int     | Minimum number of transfers in each journey  | 0           |
 | nop     | max_duration         | int     | Maximum duration of journeys in seconds (from `datetime` parameter).<br>More usefull when computing an isochrone (only `from` or `to` is provided).<br>On a classic journey (from-to), it will mostly speedup Navitia: You may have journeys a bit longer than that value (you would have to filter them).    | 86400       |
-| nop     | disruption_active    | boolean | For compatibility use only.<br>If true the algorithm take the disruptions into account, and thus avoid disrupted public transport.<br>Rq: `disruption_active=true` = `data_freshness=realtime` <br>Use `data_freshness` parameter instead       |  False     |
```

Comment:
I didn't even know this parameter existed... It doesn't work? Or we just don't want to document it?

Reply:
It existed before `data_freshness` :older_man: 
It works, I even changed a bit the way it does (see journey_common).
Only it's deprecated, so I removed it from doc... Want to get rid of the debt!

Maybe some people using it will wonder what it does: it sounds fair to let them the options of query the swagger or look at the code.